Doc/library/zipfile.rst

```diff
@@ -79,7 +79,7 @@ The module defines the following items:
    of the last modification to the file; the fields are described in section
    :ref:`zipinfo-objects`.
 
-   .. versionadded:: 3.13
+   .. versionchanged:: 3.13
       A public ``.compress_level`` attribute has been added to expose the
       formerly protected ``._compresslevel``.  The older protected name
       continues to work as a property for backwards compatibility.
```

Comment:
FYI, this attribute is only documented in this `versionadded` note; IMO, it deserves to be documented in its own `.. attribute::`.

Reply:
Let's address this in a separate PR.